catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -401,6 +424,37 @@ private String buildIngestLog(CreateRequest createReq) {
     return strBuilder.toString();
   }
 
+  /** Helper method to build file name(s) string paramater: CreateStorageRequest */
+  private String buildFileNames(CreateStorageRequest createStorageRequest) {
+    StringBuilder stringBuilder = new StringBuilder();
+
+    for (int i = 0; i < createStorageRequest.getContentItems().size(); i++) {
+      if (i > 0) {
+        stringBuilder.append(", ");
+      }
+      stringBuilder.append(createStorageRequest.getContentItems().get(i).getFilename());
+    }
+    return stringBuilder.toString();
+  }
```

Comment:
✏️ It looks like you can replace this whole method with a couple of lines, using [String.join()](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.Iterable-). You'll just need to stream the contentItems, map them to their filename, and collect to a list first. Then pass that list into the join method.
```
List<String> fileNames = createStorageRequest.getContentItems().stream().map(ContentItem::getFilename).collect(Collectors.toList());
String.join(", ", fileNames);
```

Reply:
fixed